app/models/manageiq/providers/openstack/infra_manager/refresh_parser.rb

```diff
@@ -225,6 +230,44 @@ def process_host_hardware_disks(extra_attributes)
         end
       end
 
+      def get_object_store
+        return unless @storage_service_name == :swift
+        @os_handle.service_for_each_accessible_tenant('Storage') do |svc, t|
+          svc.directories.each do |fd|
+            result = process_collection_item(fd, :cloud_object_store_containers) { |c| parse_container(c, t) }
+            process_collection(fd.files, :cloud_object_store_objects) { |o| parse_object(o, result, t) }
```

Comment:
> Btw, was that a reason why you are not storing content of the objects? 

No idea, @roliveri, do you know?  Was it just to save space until the objects were needed?